config/manager/manager.yaml

```diff
@@ -1,8 +1,6 @@
 apiVersion: v1
 kind: Namespace
 metadata:
-  annotations:
-    iks.intuit.com/managed: "false"
```

Comment:
Why are we removing this?

Reply:
@kshamajain99 Intuit specific annotations. so we shouldn't have that in this repo